pages/common/godot.md

```diff
@@ -0,0 +1,23 @@
+# godot
+
+> An open source 2D and 3D game engine.
+
+- Run a project if the current directory contains a `project.godot` file, or open the project manager:
+
+`godot`
+
+- Edit a project (the current directory must contain a `project.godot` file):
+
+`godot -e`
+
+- Open the project manager even if the current directory contains a `project.godot` file:
+
+`godot -p`
```

Comment:
Perhaps this could just be
>Open the project manager:

Reply:
The default behavior when running `godot` is to open the project manager if no `project.godot` file can be found in the current directory, so I wanted to note using `godot -p` isn't needed if the CWD doesn't have a `project.godot` file.

Reply:
Fair enough, I'd say it's fine to leave. I just wasn't sure if it was worth mentioning if using `-p` will always open it (whether or not there is a `project.godot` file).